### PR TITLE
Integrate Travis CI

### PR DESCRIPTION
**Pull request by minsooshin**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+language: node_js
+node_js:
+  - "6"
```
Comment (minsooshin) [inline .travis.yml:3]: Based on nodejs official, 6.10.2 is LTS version.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/llipio/conjugator.svg?branch=master)](https://travis-ci.org/llipio/conjugator)
```
Comment (minsooshin) [inline README.md:1]: I added this badge based on the assumption which you have travis account. I just changed reference from mine to `llipio`

File: README.md [hunk 3]
```diff
@@ -16,7 +18,10 @@ let conjugatedWord = koreanConjugator.conjugate(‘하다’, {
 ```
 
 # Tests
-`npm run test`
+```sh
+$ npm run test        # run single time
+$ npm run test:watch  # live re-run test for any change
```
Comment (minsooshin) [inline README.md:23]: The reason why I changed this is that we need single run test script for CI. If the script keeps watching the change and run test, the CI is not done.

**Comment by minsooshin** [comment]
I will try this again with non-forked repository.

**Comment by songz** [comment]
Thank you for your insightful comments @minsooshin

**Comment by minsooshin** [comment]
How it works with travis with mine. 
screen shot 2017-04-24 at 8 59 35 pm

**Comment by minsooshin** [comment]
I will open a new pull request with non-master branch

**Pull request closed without merging**